docs/shared-content/structured-configuration-variables.include.md

```diff
@@ -375,3 +375,31 @@ One limitation is that if the same prefix is declared more than once in a docume
 ```
 The namespace 'http://octopus.com' could not be mapped to the 'octopus' prefix, as another namespace 'http://octopus.com/xml' is already mapped to that prefix. XPath selectors using this prefix may not return the expected nodes. You can avoid this by ensuring all namespaces in your document have unique prefixes.
 ```
+
+## Java Properties 
+
+Given this example of a target properties file:
+
+```
+weatherApiUrl = dev.weather.com,
```

Comment:
The properties file format doesn't have commas at the end of the line. The end of the line itself is the delimiter.

```suggestion
weatherApiUrl = dev.weather.com
```

Reply:
Thankyou, im really glad you caught this. Bad copy-paste from the json example!